src/QMCDrivers/VMC/VMCBatched.cpp

```diff
@@ -292,6 +304,7 @@ bool VMCBatched::run()
     ScopedTimer local_timer(&init_walkers_timer_);
     TasksOneToOne<> section_start_task(num_crowds_);
     section_start_task(initialLogEvaluation, std::ref(crowds_));
+    section_start_task(runWarmupSteps, vmc_state, std::ref(step_contexts_), std::ref(crowds_));
```

Comment:
The init_walkers_timer is not intended for warmup. Please move it out.
if `TasksOneToOne<>` constructor is not expensive, better not to reuse.